src/common.js

```diff
@@ -252,7 +252,8 @@ export const getCompressedResources = (
   // see https://sematext.atlassian.net/browse/SC-6210
   // filter out resource types that we don't want to collect
   resources.entries = resources.entries.filter(e =>
-    !e.name.startsWith('data:') && disabledResourceTypes.indexOf(e.initiatorType) === -1);
+    !e.name.startsWith('data:') && disabledResourceTypes.indexOf(e.initiatorType) === -1
+    && !e.name.includes('rum-receiver'));
```

Comment:
Great catch! Could you also add `sematext.com` to end of `rum-receiver` to be more specific?